configuration/src/main/java/io/airlift/configuration/ConfigurationLoader.java

```diff
@@ -57,8 +60,9 @@ private ConfigurationLoader() {}
             throws IOException
     {
         Properties properties = new Properties();
-        try (InputStream inputStream = new FileInputStream(path)) {
-            properties.load(inputStream);
+        try (Reader reader = new BufferedReader(new InputStreamReader(new FileInputStream(path),
+            StandardCharsets.UTF_8))) {
```

Comment:
Please do not break the line here. Also you can use static import for UTF_8

Reply:
that's a backward incompatible change, isn't it?

what encoding does `Properties.load(InputStream)` use?

Reply:
Properties.load(InputStream) uses ISO 8859-1 Character Encoding, so it does not support multilingual encoding.

> Reads a property list (key and element pairs) from the input byte stream. The input stream is in a simple line-oriented format as specified in load(Reader) and is assumed to use the ISO 8859-1 character encoding; that is each byte is one Latin1 character. Characters not in Latin1, and certain special characters, are represented in keys and elements using Unicode escapes as defined in section 3.3 of The Java™ Language Specification.

With your help,  I checked that the multilingual value in the property file was correctly read after converting values using native2ascii.
But I think it is a good alternative to help reading multilingual encoding values in properties files without a separate conversion process.

Because it is a situation that reads the properties file, I think the backward compatibility issue will not be a problem.
How about you?

-  links
https://stackoverflow.com/questions/652161/how-do-i-convert-between-iso-8859-1-and-utf-8-in-java
https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/

Reply:
Contents of .properties file can be any `ISO 8859-1`-encoded text. 
It cannot be interpreted as UTF-8-encoded text. This may fail, or may return incorrect results.